src/xcode/ENA/ENA/Source/Scenes/TicketValidation/Cells/TicketValidationNoSupportedCertificateCellModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit.UIImage
+
+final class TicketValidationNoSupportedCertificateCellModel {
+
+	// MARK: - Init
+
+	init(
+		serviceProviderRequirementsDescription: String
+	) {
+		self.serviceProviderRequirementsDescription = serviceProviderRequirementsDescription
+	}
+
+	// MARK: - Internal
+
+	var iconImage: UIImage? {
+		return UIImage(imageLiteralResourceName: "Icon_CertificateValidation_Failed")
+	}
+
+	var serviceProviderRequirements: String? {
+		return self.serviceProviderRequirementsDescription
+	}
```

Comment:
why use computed vars here, would be here lets not enough?

Reply:
Because lets can not be used as computed property.